types/mailgun-js/mailgun-js-tests.ts

```diff
@@ -76,3 +76,20 @@ const validationResult6: mailgunFactory.validation.ValidateResponse = {
         local_part: "foo"
     }
 };
+
+// Generic requests
+mailgun.get('/samples.mailgun.org/stats', { event: ['sent', 'delivered'] }, (error, body) => {
```

Comment:
You don't need to add any logic to the callback. `(err, body) => {}` is enough.
Or `(err: Error, body: any) => {}`. That way we have type checking for the callback function.